server/src/com/thoughtworks/go/config/update/ModifyEnvironmentCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.thoughtworks.go.config.update;
+
+import com.thoughtworks.go.config.*;
+import com.thoughtworks.go.presentation.TriStateSelection;
+
+import static com.thoughtworks.go.util.ExceptionUtils.bomb;
+import static com.thoughtworks.go.util.ExceptionUtils.bombIfNull;
+
+public class ModifyEnvironmentCommand implements UpdateConfigCommand {
```

Comment:
Please mark these old classes as `@Deprecate` so we can get rid of them when the API is ready and the page is replaced with a SPA.